src/functionalTest/kotlin/io/github/reactivecircus/appversioning/tasks/GenerateAppVersionInfoTest.kt

```diff
@@ -426,6 +426,167 @@ class GenerateAppVersionInfoTest {
         }
     }
 
+    @Test
+    fun `GenerateAppVersionInfo generates versionCode and versionName from the latest git tag matching the provided tagFilter pattern when a tagFilter pattern is provided`() {
```

Comment:
@Wrywulf thanks for providing the sample tags and glob patterns. How do you feel about the scenarios demonstrated in this test?